src/js/Table/TableCell.ts

```diff
@@ -1,11 +1,27 @@
+type TableCellInitParams = {
+  table: any
+  rowData: any
+  rowIndex: number
+  columnIndex: number
+  updateRowData: any
+}
+
+type TableCellUpdateParams = {
+  table: any
+  rowData: any
+  rowIndex: number
+  columnIndex: number
+  element: HTMLElement | null
```

Comment:
- `table` should be typed `TableProps` (import from `index`)
- `rowData` should be typed `Record<string, any>`